pkg/client/manager_client.go

```diff
@@ -197,6 +197,27 @@ func (cli *BackingImageManagerClient) PrepareDownload(name, uuid string) (string
 	return resp.SrcFilePath, resp.Address, nil
 }
 
+func (cli *BackingImageManagerClient) CleanUpOrphan(name, uuid string) error {
+	if name == "" || uuid == "" {
+		return fmt.Errorf("failed to clean up orphan: missing required parameter")
+	}
+	conn, err := grpc.Dial(cli.Address, grpc.WithInsecure())
+	if err != nil {
+		return fmt.Errorf("cannot connect backing image manager service to %v: %v", cli.Address, err)
```

Comment:
Suggest using `errors.Wrapf`

Reply:
Has fixed this PR.

Will create another PR with this [issue](https://github.com/longhorn/longhorn/issues/5015) for all the error logging in this repo since the scope is large.